docs/cli-roadmap.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes CLI/Configuration Roadmap
+
+See also issues with the following labels:
+* [area/config-deployment](https://github.com/GoogleCloudPlatform/kubernetes/labels/area%2Fconfig-deployment)
+* [component/CLI](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2FCLI)
+* [component/client](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2Fclient)
+
+At least the first three issues should block 1.0.
+
+1. stop command in kubectl [#2144](https://github.com/GoogleCloudPlatform/kubernetes/issues/2144): Gracefully terminate any object(s), to the extent currently possible. We also eventually want first-class support in the API for graceful termination. [#1535](https://github.com/GoogleCloudPlatform/kubernetes/issues/1535)
+1. Kind-based filtering on object streams -- only operate on the kinds of objects specified
```

Comment:
This is not at all about scalability. It's to make operations on groups of objects of multiple kinds, specified via file or directory, much more useful. It could, for example, be used as a workaround for the next issue, by specifying that just services should be created.